ansible/roles/test/files/ptftests/fib_test.py

```diff
@@ -173,17 +174,19 @@ def check_ip_range(self, ip_range, next_hop, ipv4=True):
         # Send a packet with a random IP in the range
         if ip_range.length() > 2:
             self.check_ip_route(src_port, ip_range.get_random_ip(), exp_port_list, ipv4)
-
+        time.sleep(0.01)
```

Comment:
Why need this delay?

If this delay is really needed, should it be added to the loop in function `def check_ip_ranges`?

```python
    def check_ip_ranges(self, ipv4=True):
        if ipv4:
            ip_ranges = self.fib.ipv4_ranges()
        else:
            ip_ranges = self.fib.ipv6_ranges()

        for ip_range in ip_ranges:
            if ip_range.get_first_ip() in self.fib:
                next_hop = self.fib[ip_range.get_first_ip()]
                self.check_ip_range(ip_range, next_hop, ipv4)
                time.sleep(0.01)
```

Reply:
Yes, we need this tiny sleep to avoid flapping bgp sessions, as TC received ~6k routes per neighbor, and sends 3 packets per route group. To avoid overloading when send traffic delay was being set. 

In def check_ip_ranges i beleive we do not need such delay, I've re-run fib dozen times and TC passed without changes in check_ip_ranges.